code/Ui/Win32/PathDialogWin32.cpp

```diff
@@ -14,42 +14,68 @@ namespace traktor::ui
 {
 
 PathDialogWin32::PathDialogWin32(EventSubject* owner)
-:	m_owner(owner)
+	: m_owner{ owner }
```

Comment:
Is there a point with using curly braces instead of parenthesis here?
List initialization doesn't do anything here from what I understand?
Also, I prefer initializing members in class declaration instead just to skip the hazzle of C++ member initialization order in definition.